apis/kueue/v1beta1/provisioningrequestconfig_types.go

```diff
@@ -57,6 +57,53 @@ type ProvisioningRequestConfigSpec struct {
 	// +listType=set
 	// +kubebuilder:validation:MaxItems=100
 	ManagedResources []corev1.ResourceName `json:"managedResources,omitempty"`
+
+	// retryStrategy defines strategy for retrying ProvisioningRequest.
+	// If null, then the default configuration is applied. To switch off retry mechanism
+	//  set retryStrategy.backoffLimitCount to 0.
+	//
+	// +optional
+	RetryStrategy *ProvisioningRequestRetryStrategy `json:"retryStrategy,omitempty"`
+}
+
+var (
+	DefaultRetryStrategy = ProvisioningRequestRetryStrategy{
+		BackoffLimitCount:  nil,
+		BackoffBaseSeconds: 60,   // 1 min
+		BackoffMaxSeconds:  1800, // 30 min
+	}
+)
+
+type ProvisioningRequestRetryStrategy struct {
+	// BackoffLimitCount defines the maximum number of re-queuing retries.
+	// Once the number is reached, the workload is deactivated (`.spec.activate`=`false`).
+	// When it is null, the workloads will repeatedly and endless re-queueing.
+	//
+	// Every backoff duration is about "b*2^(n-1)+Rand" where:
+	// - "b" represents the base set by "BackoffBaseSeconds" parameter,
+	// - "n" represents the "workloadStatus.requeueState.count",
+	// - "Rand" represents the random jitter.
+	// During this time, the workload is taken as an inadmissible and
+	// other workloads will have a chance to be admitted.
+	// By default, the consecutive requeue delays are around: (60s, 120s, 240s, ...).
+	//
+	// +optional
+	BackoffLimitCount *int32 `json:"backoffLimitCount,omitempty"`
+
+	// BackoffBaseSeconds defines the base for the exponential backoff for
+	// re-queuing an evicted workload.
+	//
+	// Defaults to 60.
+	// +optional
+	// +kubebuilder:default=60
+	BackoffBaseSeconds int32 `json:"backoffBaseSeconds,omitempty"`
+
+	// BackoffMaxSeconds defines the maximum backoff time to re-queue an evicted workload.
+	//
+	// Defaults to 1800.
+	// +optional
```

Comment:
```suggestion
	// +required
```

Reply:
Should it be required if we default this value with kubebuilder annotation?

Reply:
I this is independent thing,

Reply:
We should also drop omitempty for required fields

Reply:
Done